packages/rich-text/src/plugins/List/index.tsx

```diff
@@ -175,16 +176,15 @@ const hasListAsDirectParent = (editor: Editor, path: Path) => {
   return isList(parentNode as CustomElement);
 };
 
-const isValidInsideList = (node: TextOrCustomElement) =>
+const isValidInsideListItem = (node: TextOrCustomElement) =>
   Text.isText(node as TextElement) ||
   (LIST_ITEM_BLOCKS as Array<TopLevelBlockEnum | INLINES>)
     .concat(Object.values(INLINES))
     .includes((node as CustomElement).type as TopLevelBlockEnum);
 
 const replaceInvalidListItemWithText = (editor: PlateEditor, path: Path) => {
-  const textFromEntry = slateNodeEntryToText(editor, path);
-  Transforms.removeNodes(editor, { at: path });
-  Transforms.insertNodes(editor, textFromEntry, { at: path });
+  const textFromEntry = extractParagraphsAt(editor, path);
```

Comment:
Nit: Seems like whenever we use `extractParagraphsAt` it's followed by a `replaceNode`, wondering if it would be more slightly more efficient if the function would already do that (assuming there's a slate helper that can be used instead of doing the current `Editor.nodes` call).

Reply:
I couldn't find a better way